src/wallet/wallet.cpp

```diff
@@ -1948,6 +1951,8 @@ void CWallet::ResendWalletTransactions()
         auto locked_chain = chain().lock();
         LOCK(cs_wallet);
 
+        if (m_is_ibd) return;
```

Comment:
Perhaps add this change in the commit "Add m_is_ibd in CWallet`

Reply:
re: https://github.com/bitcoin/bitcoin/pull/17484#discussion_r347356956

In commit "Remove Chain::isReadyToBroadcast method" (a5a674857d0ae6d535e4725337968492c821c635)

> Perhaps make this change in the commit "Add m_is_ibd in CWallet` (or merge the two commits, IIUC)

Change does make sense (and is necessary) in this commit due to isReadyToBroadcast call a few lines up. But it would be good to add a comment here mentioning this is also needed to avoid spamming nodes with old transactions.